### PR TITLE
Migrations: Update IDTool versions

### PR DESCRIPTION
**Pull request by Hwesta**
Update FIDO & Siegfried versions in the FPR to match what will be installed.

The fpr-admin submodule commit in Archivematica will also have to be updated once this is merged.

### CODE REVIEW

File: fpr/migrations/0005_update_tool_versions.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+from django.db import migrations, models
+
+
+def data_migration(apps, schema_editor):
+    IDTool = apps.get_model('fpr', 'IDTool')
+    IDTool.objects.filter(description='Fido', version='1.3.4').update(version='1.3.5')
+    IDTool.objects.filter(description='Siegfried', version='1.5.0').update(version='1.6.6')
+
+def reverse_migration(apps, schema_editor):
```
Comment (sevein) [inline fpr/migrations/0005_update_tool_versions.py]: Nice to see this!

**Review by sevein — APPROVED** [review]

**Comment by sevein** [comment]
It seems that the latest release of Siegfried is `v1.6.7`.

**Comment by Hwesta** [comment]
I ninja-updated the PR to say 1.6.7 already. :) I'll check this is correct once we know what version of Siegfried we're actually updating to, but I suspect it will be 1.6.7

**Comment by jrwdunham** [comment]
👍 Confirmed that this change updates the Preservation Planning tab of Archivematica so that Fido's version is listed as 1.3.5 and Siegfried's is listed as 1.6.7. (On my VM running qa/1.x and qa/0.x, Fido is at 1.3.5 and Siegfried is at 1.5.0.)

**Comment by Hwesta** [comment]
Submodule commit update in Archivematica qa/1.x as 190bc9140245edaa24c4328cb05e6b868b9052b1 and qa/1.6.x as 68f1380c651c4b3979963f6b1ff051f619961103

**Pull request merged**